docs/migration-guides/v2/lib.md

```diff
@@ -0,0 +1,284 @@
+# Migration guide for ably-js v2
+
+Here’s how to migrate from ably-js v1 to v2:
+
+1. [Stop using functionality that v1 deprecated and which v2 removes](#stop-using-v1-deprecated).
+1. [Stop using other functionality that v2 removes](#stop-using-v2-removed).
+1. [Update to v2 and handle its breaking changes](#update-to-v2-and-handle-breaking-changes).
+1. (Optional) [Stop using functionality that v2 deprecates](#stop-using-v2-deprecated).
+1. (Optional) [Take advantage of new features that v2 introduces](#use-v2-new-features).
```

Comment:
it's probably temporary while WIP, but commenting to not forget change numbers to 1, 2, 3 ...

Reply:
Not sure if you know, but Markdown actually doesn't care about the numbers that you write in your source for an ordered list. This still gets rendered as 1, 2, 3, …, and you don't have to care about keeping the numbers up to date in your source as you insert / remove / reorder list items.

Reply:
I actually didn't know this, and find it quite strange 😅 i do think one of the key advantages of markdown is that the source is human readable so you don't need a renderer (and i rarely, if ever, use one locally), so i marginally prefer using an actual sequence here

Reply:
Sure, have changed

Reply:
Same, didn't know about this feature, I'm too used to IDEs automatically populating those lists now, had no idea markdown does it in the back too.
Anyway, it's changed now, closing conversation.